bentoml/repository/__init__.py

```diff
@@ -147,11 +153,31 @@ def add(self, bento_name, bento_version):
 
     def get(self, bento_name, bento_version):
         # Return s3 path containing uploaded Bento files
-        raise NotImplementedError
+
+        object_name = self._get_object_name(bento_name, bento_version)
+
+        try:
+            response = self.s3_client.generate_presigned_url(
+                'get_object',
+                Params={'Bucket': self.bucket, 'Key': object_name},
+                ExpiresIn=self._expiration,
+            )
+        except Exception as e:
+            raise BentoMLRepositoryException(
+                "Not able to get pre-signed URL on S3. Error: {}".format(e)
+            )
+        return response
 
     def dangerously_delete(self, bento_name, bento_version):
         # Remove s3 path containing related Bento files
-        raise NotImplementedError
+
+        object_name = self._get_object_name(bento_name, bento_version)
+
+        response = self.s3_client.delete_object(Bucket=self.bucket, Key=object_name)
+
+        DELETE_MARKER = 'DeleteMarker'
```

Comment:
add a comment about this field in boto3's response and what it means?

Reply:
ok